samples/sampler/stories/qa/VideoPlayer.js

```diff
@@ -97,3 +99,50 @@ export default {
 export const PreloadVideos = () => <VideoSourceSwap />;
 
 PreloadVideos.storyName = 'Preload Videos';
+
+class VideoPlayerWithfastForwardMode extends Component {
+	constructor (props) {
+		super(props);
+	}
+
+	setVideoPlayer = (node) => {
+		this.videoPlayer = node;
+	};
+
+	fastforward = () => {
+		this.videoPlayer.fastForward();
+	};
+
+	render () {
+		return (
+			<div>
+				<VideoPlayer
+					feedbackHideDelay={0}
+					muted
+					playbackRateHash={{
+						fastForward: [1.25, '3/2', '2', '2.5', '4', '8'],
+						rewind: ['-2', '-4', '-8', '-16'],
+						slowForward: ['1/4', '1/2'],
+						slowRewind: ['-1/2', '-1']
+					}}
+					ref={this.setVideoPlayer}
+					title={'Big Buck Bunny'}
+				>
+					<Video>
+						<source src={'http://clips.vorwaerts-gmbh.de/big_buck_bunny.mp4'} />
+					</Video>
+					<MediaControls actionGuideLabel="Press down button to fastforward">
+						<Button
+							icon="forward"
+							onClick={this.fastforward}
+						/>
+					</MediaControls>
```

Comment:
Could you please add other buttons for "rewind" , "slowForward", and "slowRewind"?

Reply:
I added rewind button. "slowForward" and "slowRewind" can be tested when video is paused and fastforward or rewind button is clicked.